src/modules/app/actions/init-augur.js

```diff
@@ -97,8 +97,8 @@ function loadAccount(dispatch, existing, accountType, callback) {
         loggedInAccount !== account
       ) {
         // local storage does not match mm account
-        dispatch(logout());
-        account = null;
+        dispatch(useUnlockedAccount(account));
+        loggedInAccount = account;
```

Comment:
also, for hosted environment it still logs you out of ui when you switch mm accounts. need to switch logout func in next few if/else statement as well